projects/admin/src/app/service/record-permission.service.ts

```diff
@@ -196,7 +196,8 @@ export class RecordPermissionService {
     return {
       is_default: this._translateService.instant('The default record cannot be deleted'),
       has_settings: this._translateService.instant('The record contains settings'),
-      harvested: this._translateService.instant('The record has been harvested')
+      harvested: this._translateService.instant('The record has been harvested'),
+      already_deleted_status: this._translateService.instant('This item has already the deleted status')
```

Comment:
The message is not really user friendly. You can have for example: 'This item is already deleted' or similar.

Reply:
done